src/transformStateWithClones.js

```diff
@@ -7,7 +7,38 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const newState = {
```

Comment:
```suggestion
  const stateClone = {
```
do not use words like 'new' in variable names